src/components/DryRunnerHand/index.js

```diff
@@ -13,7 +13,7 @@ export default React.memo(function DryRunnerHand(props) {
   return (
     <Row data-testid='hand' desktopOnly>
       {chunk(paddedHand, 2).map(([cardA, cardB], index) => (
-        <Column key={cardA || index}>
+        <Column key={cardA ? cardA.id + '#' + cardA.idx : index}>
```

Comment:
```suggestion
        <Column key={cardA ? cardA.id + '_' + cardA.idx : index}>
```
No reason not to be consistent. :)